src/networks-of-brokers.md

```diff
@@ -406,9 +406,11 @@ For example, if using distributed queues, you may wish to have equivalent weight
 **N.B.** You can only use [wildcards](wildcards) in the `excludedDestinations` and `dynamicallyIncludedDestinations` properties.  
 **N.B.** **Do not** change the name of the bridge or the name of the Broker if you are using durable topic subscribers across the network. Internally ActiveMQ uses the network name and broker name to build a unique but repeatable durable subscriber name for the network.
 
-### Stuck Messages (version 5.6)
+### Stuck Messages
```

Comment:
Maybe worth to mention TTL between brokers in the NoB ?

Reply:
I don't see a section about quickly describing consumer TTL and TTL generally speaking. If you are not confortable about this, no worries, I will do a section about that.

Reply:
I can try to initiate a section from the Tim comment in the Mailing list.